volttron/platform/auth.py

```diff
@@ -427,6 +430,14 @@ def deny_authorization_failure(self, user_id):
                     self._auth_denied.append(pending)
                     del self._auth_failures[self._auth_failures.index(pending)]
 
+        if len(self._auth_approved) > 0:
```

Comment:
This if statement shouldn't be necessary

Reply:
Sounds good. They have been removed!